lib/block_view_association.go

```diff
@@ -0,0 +1,857 @@
+package lib
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+const MaxAssociationTypeCharLength int = 64
+const MaxAssociationValueCharLength int = 256
+const AssociationTypeReservedPrefix = "DESO"
+const AssociationQueryWildcardSuffix = "*"
```

Comment:
I'm happy to move these constants elsewhere like lib/constants.go, but they're association-specific constants so thought here was also a suitable place. LMK.